packages/react-widgets/src/MultiselectInput.tsx

```diff
@@ -34,9 +34,9 @@ const MultiselectInput = React.forwardRef(
       />
     )
   },
-)
+);
 
-MultiselectInput.displayName = 'MultiselectInput'
-MultiselectInput.propTypes = propTypes
+MultiselectInput.displayName = 'MultiselectInput';
+(MultiselectInput as any).propTypes = propTypes;
```

Comment:
this is confusing, components should allow propTypes

Reply:
in this case is fixed.

For other cases like `NumericPicker` its more complicated, requiring changing:
`function NumberPicker(uncontrolledProps: NumberPickerProps) {`
to
`const NumberPicker : StatelessComponent<NumberPickerProps> = function NumberPicker(uncontrolledProps: NumberPickerProps) {`

then 
`const propTypes = {`
to
`const propTypes :WeakValidationMap<NumberPickerProps> = {`

I can fix `format: CustomPropTypes.numberFormat,` by using `createChainableTypeChecker<string>(`

but I don't know how to fix `localizer: PropTypes.object,` without changing it to `localizer: PropTypes.any,`, so I roll-backed.

Reply:
I should also mention that in the cases of the main widgets the prop-types and defaultProps were intentionally not typed as prop-types, b/c the the inference got very complicated and ended up either breaking the generics or being wrong. I think it's fine to hack around that if that's what this is. I don't actually find the clever prop-types type matching the react types do very helpful

Reply:
Agree, sometimes TypeScrip definitions get crazy complicated. I prefer an any to having to get a PhD to fix a build error.